### PR TITLE
🤷🏿 Reduce number of locations shown

### PR DESCRIPTION
**Pull request by Qrtn**
## Status:
:rocket: Ready

## Description

Reduces some re-renders, and changes max number of locations displayed on map from 1000 to 250 for now.

### CODE REVIEW

**Review by laurenho025 — APPROVED** [review]
lgtm 🚀

File: src/components/map/Cluster.js [hunk 1]
```diff
@@ -58,11 +59,13 @@ const calculateDiameter = (count) =>
  * @param {function} onClick - The handler called when this cluster is clicked
  * @param {number} count - The total number of locations within this cluster
  */
-const Cluster = ({ onClick, count }) => (
+const Cluster = memo(({ onClick, count }) => (
```
Comment (SirajChokshi) [inline src/components/map/Cluster.js:62]: Nice!

File: src/components/map/MapPage.js [hunk 2]
```diff
@@ -62,13 +64,13 @@ const MapPage = () => {
           swlng: bounds.sw.lng,
           muni: filters.muni ? 1 : 0,
           t: filters.types.toString(),
+          limit: 250,
```
Comment (SirajChokshi) [inline src/components/map/MapPage.js:67]: should we move this constant `{...limit: 250}` to a utils file or is keeping it local good for now?

**Review by SirajChokshi — APPROVED** [review]
Great work! 🤩

File: src/components/map/MapPage.js (hunk 2 above)
Comment (Qrtn) [reply src/components/map/MapPage.js:113]: Will eventually consolidate all getLocations requests in a file in utils!

**Pull request merged**